src/main/kotlin/uk/gov/justice/digital/hmpps/prisonertonomisupdate/locations/LocationsApiService.kt

```diff
@@ -55,4 +56,14 @@ class LocationsApiService(
       .retryWhen(backoffSpec.withRetryContext(Context.of("api", "locations-api", "url", url.path)))
       .awaitSingle()
   }
+
+  suspend fun patchNonResidentialLocation(id: String, request: PatchNonResidentialLocationRequest) {
+    webClient
+      .patch()
+      .uri("/locations/non-residential/{id}", id)
+      .bodyValue(request)
+      .retrieve()
+      .awaitBodilessEntity()
+    // there is a body, the modified Location, but ignore it
```

Comment:
Why not just return it and ignore in the calling class? Then you don't need a comment to explain strangeness as it's not strange.